app/move/controllers/view/view.update.urls.js

```diff
@@ -0,0 +1,17 @@
+const getUpdateUrls = (updateSteps, moveId, canAccess) => {
```

Comment:
I think rather than passing in the `canAccess` function which is a view helper it would be cleaner to pass in the current user's permissions and use the [permissions module](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/5f692a046faea2298687c7a23bc4882734c7a60b/common/middleware/permissions.js#L3) to do this check.

Reply:
Done